source/Coop.IntegrationTests/Settlement/SettlementChangeOwnerTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Coop.IntegrationTests.Environment;
+using GameInterface.Services.Settlements.Messages;
+
+namespace Coop.IntegrationTests.Settlement
+{
+    public class SettlementChangeOwnerTest
+    {
+        internal TestEnvironment TestEnvironment { get; }
+
+        public SettlementChangeOwnerTest()
+        {
+            // Creates a test environment with 1 server and 2 clients by default
+            TestEnvironment = new TestEnvironment();
+        }
+
+        /// <summary>
+        /// Verify sending StartSettlementEncounterAttempted on one client
+        /// Triggers PartyEnterSettlement on all other clients
+        /// </summary>
+        [Fact]
+        public void SettlementChangeOwner_Publishes_AllClients()
+        {
+            // Arrange
+            var settlementId = "Test Settlement";
+            var ownerId = "Owner Id";
+            var capturerId = "Capturer Id";
+            var detail = 2;
+
+            var message = new LocalSettlementOwnershipChange(settlementId, ownerId, capturerId, detail);
```

Comment:
Integration tests shouldn't use any game interface messages. These tests are meant to test the protocal between the client, server, and other clients. In this case this message should be `SettlementOwnershipChangeRequest` and it should check that the server receives that event and the other clients receive `SettlementOwnershipChangeApproved`